### PR TITLE
added callouts

### LINKED ISSUE #524: [Site] Callout to Discussion Forum
**Current State of the Meshery.io site**
meshery.io is missing prominent links to the service mesh discussion forum.

**Desired State of the Meshery.io site**
Akin to the recent callout to the service mesh discussion forum provided on layer5.io/blog, add one or more calls to action on meshery.io

---
### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/layer5io/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabh1908@gmail.com>

**Description**
Added callouts to various resources in place of the list present previously.
This PR fixes #524 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
Screenshot_20210829-023856

Looks great!, But a little too big for mobile view, could we get it a little smaller.?

**Review by warunicorn19 — DISMISSED** [review]
LGTM 🎉, thanks @debo19

**Review by Tanuj22 — DISMISSED** [review]
LGTM

File: _includes/partials/callouts.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<div class="callout">
+  <a class="card1" href="https://bit.ly/2SbrRhe">
+    <h3>Community Calendar</h3>
+    <p class="small">Join the weekly meetings by subscribing to our community calendar</p>
+    <div class="go-corner" href="https://bit.ly/2SbrRhe">
+      <div class="go-arrow">
+        <img id="logo_meshery" src="{{ site.baseurl }}/images/logos/meshery-logo.png" alt="" />
+      </div>
```
Comment (Nikhil-Ladha) [inline _includes/partials/callouts.html:8]: Considering you are already adding the icon using CSS just above this line.
I don think we need the image being added separately. The same is applicable for the code below.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
A small nitpick: is it possible have all of the cards of equal height (size)?

**Review by leecalcote — APPROVED** [review]

**Comment by debo19** [comment]
cards
@leecalcote  They seem to have the same height for me, is it different for you?

**Comment by warunicorn19** [comment]
Screenshot 2021-08-31 at 11 21 11 AM

safari doing it's thing again.

File: _includes/partials/callouts.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<div class="callout">
+  <a class="card1" href="https://bit.ly/2SbrRhe">
+    <h3>Community Calendar</h3>
+    <p class="small">Join the weekly meetings by subscribing to our community calendar</p>
+    <div class="go-corner" href="https://bit.ly/2SbrRhe"></div>
+  </a>
+   <a class="card1" href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA">
+    <h3>Community Meetings</h3>
+    <p class="small">Watch community meeting recordings on our youtube channel</p>
```
Comment (leecalcote) [inline _includes/partials/callouts.html:9]:
```suggestion
    <p class="small">Watch community meeting recordings on our YouTube channel</p>
```

File: _includes/partials/callouts.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<div class="callout">
+  <a class="card1" href="https://bit.ly/2SbrRhe">
+    <h3>Community Calendar</h3>
+    <p class="small">Join the weekly meetings by subscribing to our community calendar</p>
+    <div class="go-corner" href="https://bit.ly/2SbrRhe"></div>
+  </a>
+   <a class="card1" href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA">
```
Comment (leecalcote) [inline _includes/partials/callouts.html:7]: YouTube links need to include the subscription confirmation. 

```suggestion
   <a class="card1" href="https://www.youtube.com/c/Layer5io?sub_confirmation=1">
```

File: _sass/landing-page.scss [hunk 4]
```diff
@@ -304,26 +301,131 @@
...
+    margin-top: 0.5rem;
+    margin-right: 0.5rem;
+    width: 31px;
+    height: 31px;
+    overflow: hidden;
+    top: 0;
+    right: 0;
+    background-color: none;
+    border-radius: 0 4px 0 3px;
+    content: url('../images/logos/meshery-logo.png');
```
Comment (leecalcote) [inline _sass/landing-page.scss:373]: As a general rule, let's prefer SVGs to PNGs.

**Review by leecalcote — APPROVED** [review]
Thank you, @debo19. 🥇

**Pull request merged**